openshift/e2e-tests-openshift.sh

```diff
@@ -13,10 +13,14 @@ MAX_NUMBERS_OF_PARALLEL_TASKS=4
 KUBECTL_CMD="kubectl --cache-dir=/tmp/cache"
 
 # Give these tests the priviliged rights
-PRIVILEGED_TESTS="buildah buildpacks buildpacks-phases jib-gradle kaniko kythe-go s2i"
+PRIVILEGED_TESTS="buildah buildpacks buildpacks-phases jib-gradle kaniko kythe-go orka-init orka-teardown s2i"
 
 # Skip those tests when they really can't work in OpenShift
-SKIP_TESTS="docker-build orka-full"
+SKIP_TESTS="docker-build orka-full orka-deploy"
+
+# Orka Tasks which can be tested as privileged but existing SA needs to be used
+# to give privileged access
+ORKA_TASKS="orka-init orka-teardown"
```

Comment:
I don't like much these kinds of specifics tasks, to be honest i would not be against just disabling it, (it's not like it's something critical for openshift pipeline product)